tests/integrationv2/test_well_known_endpoints.py

```diff
@@ -109,13 +113,17 @@ def test_well_known_endpoints(managed_process, protocol, endpoint, provider, cip
 
     # expect_stderr=True because S2N sometimes receives OCSP responses:
     # https://github.com/aws/s2n-tls/blob/14ed186a13c1ffae7fbb036ed5d2849ce7c17403/bin/echo.c#L180-L184
-    client = managed_process(provider, client_options, timeout=5, expect_stderr=True)
+    client = managed_process(provider, client_options, timeout=60, expect_stderr=True)
 
     expected_result = EXPECTED_RESULTS.get((endpoint, cipher), None)
 
     for results in client.get_results():
         results.assert_success()
 
+    if provider is CriterionS2N and provider.criterion_mode is CriterionS2N.criterion_report:
+        # In report mode, criterion doesn't actually run the test, so no output to check.
```

Comment:
In report mode it does all the setup and initialization of the benchmark, but then just compares the json data and exits. Moving the report run to a cleanup method does sound cleaner.

Reply:
upon reflection, not sure how this would work- the rerun of the test would have to be initiated at the tox/pytest level.  punting on this for now.

Reply:
> I still have the same comment after looking at this again, I just don't love having a report mode run that needs to  branch here. Just wanted to see if you've thought of a way around this since then.

Essentially we need to run the test twice with different flags, didn't see a clear direct way to accomplish this.

> I think I don't understand what Criterion is doing in report mode. What do you mean by "doesn't actually run the test"? Isn't it running the test since it reaches this point where you have to branch?

What I observed was that when called with the "report" flags, criterion doesn't run `s2nc`, so the output required to pass the test doesn't exist. Not seeing a direct way around this, no.

Reply:
More experimentation led to a way to get this out- there is no more report mode.